app/views/lookup_keys/_fields.html.erb

```diff
@@ -14,36 +14,37 @@
     <h2><%= _("Default behavior") %></h2>
     <h6><%= _("Override the default value of the Puppet class parameter.") if is_param%></h6>
     <%= checkbox_f(f, :override, :onchange => 'toggleOverrideValue(this)', :size => "col-md-8",
-                   :help_inline => popover('', _('Whether the class parameter value is managed by Foreman.'))
+                   :label_help => _('Whether the class parameter value is managed by Foreman.')
         ) if is_param %>
     <%= param_type_selector(f, :onchange => 'keyTypeChange(this)') %>
     <%= textarea_f f, :default_value, :value => f.object.default_value_before_type_cast, :size => "col-md-8",
                    :disabled => (is_param && (!f.object.override || f.object.omit)),
                    :input_group_btn => fullscreen_input,
                    :rows => 1,
-                   :help_inline => popover('', _("Value to use when there is no match.")),
+                   :label_help => _("Value to use when there is no match."),
                    :class => "no-stretch #{'masked-input' if f.object.hidden_value?}" %>
     <div class="form-group">
       <%= checkbox_f(f, :omit, :label => _('Omit'),
-                     :help_inline => omit_help,
+                     :label_help => omit_help_body,
+                     :label_help_options => { :title => omit_help_title },
                      :size => "col-md-1", :label_size => "col-md-2", :table_field => true,
                      :onchange => 'toggleOmitValue(this, "default_value")',
                      :disabled => (is_param && !f.object.override)) if is_param %>
       <%= checkbox_f(f, :hidden_value, :label => _('Hidden value'),
                      :class => 'hidden_value_textarea_switch', :onchange => 'toggle_lookupkey_hidden(this)',
                      :checked => f.object.hidden_value?,
-                     :help_inline => popover("", _("Hide all values for this parameter.")),
+                     :label_help => _("Hide all values for this parameter."),
                      :size => "col-md-1", :label_size => "col-md-2", :table_field => true,
                      :disabled => (is_param && !f.object.override)) %>
     </div>
   </fieldset>
   <fieldset>
     <%= collapsing_header _("Optional input validator"), "#optional_input_validators_#{f.object.id}", "collapsed" %>
     <div id="optional_input_validators_<%= f.object.id %>" class="collapse out">
-      <%= alert :class => 'alert-info', :header => _('Note'), :text => _('If ERB is used in a parameter value, the validation of the value will happen during the ENC request. If the value is invalid, the ENC request will fail.') %>
+      <h6><%= _('If ERB is used in a parameter value, the validation of the value will happen during the ENC request. If the value is invalid, the ENC request will fail.') %></h6>
```

Comment:
No, but it brings consistency with the role form too, it's the same thing that @rohoover suggested there.